package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "gonzaga-access.github.io",
+  "version": "1.0.0",
+  "description": "You can use the [editor on GitHub](https://github.com/GonzagaAccess/gonzaga-access.github.io/edit/master/README.md) to maintain and preview the content for your website in Markdown files.",
+  "main": "index.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/parcel src/index.html",
```

```suggestion
    "dev": "parcel src/index.html",
```

You don't need set based on node_modules, therefore `npm` already recognize it